controller/space.go

```diff
@@ -126,14 +126,17 @@ func (c *SpaceController) Create(ctx *app.CreateSpaceContext) error {
 	}
 
 	// Create keycloak resource for this space
-	resource, err := c.resourceManager.CreateResource(ctx, ctx.RequestData, spaceID.String(), spaceResourceType, &spaceName, &scopes, currentUser.String())
+
+	resource, err := c.resourceManager.CreateSpace(ctx, ctx.RequestData, spaceID.String())
```

Comment:
goasupport.ForwardContextRequestID(ctx) ?

Reply:
goasupport.ForwardContextRequestID(ctx) is used inside resourceManager.Create/DeleteSpace()